0-bootstrap/modules/jenkins-agent/variables.tf

```diff
@@ -86,22 +127,23 @@ variable "project_prefix" {
 }
 
 variable "activate_apis" {
-  description = "List of APIs to enable in the Cloudbuild project."
+  description = "List of APIs to enable in the CICD project."
   type        = list(string)
 
   default = [
     "serviceusage.googleapis.com",
     "servicenetworking.googleapis.com",
     "compute.googleapis.com",
     "logging.googleapis.com",
-    "bigquery.googleapis.com",
+    "bigquery.googleapis.com", // required to create the BQ log sinks
     "cloudresourcemanager.googleapis.com",
     "cloudbilling.googleapis.com",
     "iam.googleapis.com",
     "admin.googleapis.com",
     "appengine.googleapis.com",
     "storage-api.googleapis.com",
-    "cloudkms.googleapis.com"
+    "cloudkms.googleapis.com",
+    "sourcerepo.googleapis.com",
```

Comment:
do we need source repos? My assumption was the Jenkins is targeting users with other git hosting solutions anyway.

Reply:
Good catch. removing this and kms as well.